From bc5d3bf623e3f421ad301453a458b65b0bb1598e Mon Sep 17 00:00:00 2001
From: Aditya Kumar Singh <quic_adisi@quicinc.com>
Date: Mon, 12 Aug 2024 14:32:53 +0530
Subject: [PATCH] tests: Enable MLD GTK rekey test

Now that MLO GTK rekey support is added, enable the test case fully.

Signed-off-by: Aditya Kumar Singh <quic_adisi@quicinc.com>
---
 tests/hwsim/test_eht.py | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/tests/hwsim/test_eht.py b/tests/hwsim/test_eht.py
index 97922152d..5ce825870 100644
--- a/tests/hwsim/test_eht.py
+++ b/tests/hwsim/test_eht.py
@@ -757,10 +757,9 @@ def test_eht_mld_gtk_rekey(dev, apdev):
             if "CTRL-EVENT-DISCONNECTED" in ev:
                 raise Exception("Disconnect instead of rekey")
 
-            #TODO: Uncomment these ones GTK rekeying works for MLO
-            #time.sleep(0.1)
-            #traffic_test(wpas, hapd0)
-            #traffic_test(wpas, hapd1)
+            time.sleep(0.1)
+            traffic_test(wpas, hapd0)
+            traffic_test(wpas, hapd1)
 
 def test_eht_ml_probe_req(dev, apdev):
     """AP MLD with two links and non-AP MLD sending ML Probe Request"""
-- 
2.43.0

